### DISCUSSION TITLE
What am i doing wrong?

### DISCUSSION THREAD

**Discussion by ninjapigeon** [remotely-save/remotely-save#777] [Q&A] [answered]
Created a vault, added some files and folders. (On iPad)
Turned on Remotely Save, connected my pCloud account. Shows good there, but no progress bar or other visual indication beyond the 'disconnect' button to confirm.
Did not pre-create any folders on my pCloud account, yet regardless of what option i seem to enable in remotely-save, nothing seems to sync / backup to my pCloud.
Also enabled remotely-save on my iPhone with a new vault named the same as the supposedly 'synced' vault, and despite also logging in, connecting, nothing syncs down.

What am i missing?

**Comment by ninjapigeon** [comment, marked as answer]
Finally figured it out. For pCloud synching anyway.
After choosing your cloud service, then scroll down to sign in/up for remotely-save, then copy that auth code...
The top of the page has an Auth section with a tiny "auth" word below it.
Apparently this does *not* mean it is authorized, it means you have to click "Auth" to authorize with your cloud service (in this case pCloud).
Did that, and poof - the folder appeared on my pCloud drive, and looks like it has all synced over!

Now to just get them to sync down on my iPhone...

**Comment by fyears** [comment]
Thanks for sharing.

> The top of the page has an Auth section with a tiny "auth" word below it.

May I ask which page do you mean? if that’s remotely save’s part, I could try to optimize it.
